src/components/LogViewer/LogViewer.tsx

```diff
@@ -102,6 +105,23 @@ export const LogViewer = ({ isStatic = false, data, heightOffset = 0, customActi
     return () => clearTimeout(timeoutId);
   }, []);
 
+  useEffect(() => {
+    const handleKeyDown = (event: KeyboardEvent) => {
+      if (event.key === '/') {
+        event.preventDefault();
+        const searchInput = document.querySelector<HTMLInputElement>(
```

Comment:
Did you consider using `ref={ref}` attribute?

```ts
const ref: React.MutableRefObject<HTMLInputElement | null> = React.useRef(null);
```

Reply:
I tried but we can't use ref to access the internal input element of LogViewerSearch because LogViewerSearch from PF does not support ref forwarding or provide a way to pass a ref to the internal SearchInput component.

Reply:
I've checked the implementation details and I think it would be passed automatically.

Reply:
After getting deep into the PF search bar components, the SearchInput component and its underlying components do not forward the ref all the way down to the actual <input> element.

Even though SearchInput accepts a ref prop and passes it as innerRef to SearchInputBase, and then to TextInputGroupMain, the chain breaks because TextInputGroupMain does not pass innerRef to the underlying <input>. See the following code from PF: https://github.com/patternfly/patternfly-react/blob/a3ffb39a0cc2c9130f7db86ad3186787ba12648e/packages/react-core/src/components/SearchInput/SearchInput.tsx#L292-L305

So I believe we have to use `document.querySelector` under this circumstance. @matedo1